src/server/server.js

```diff
@@ -33,7 +33,7 @@ const startServer = (port, ssoEndpoint, apigeeEndpoint, basePath) => {
         if (url.indexOf('/token') === 0) {
             state.getToken().subscribe(
                 data => { res.end(data); },
-                error => { res.end(error); }
+                error => { res.statusCode = 400; res.end(error.message); }
```

Comment:
If the error handler is called, why it should be a 400 (user error)? shouldn't this be considered a 500 (server error)?